lib/commands/actions.js

```diff
@@ -1,14 +1,14 @@
-import androidHelpers from '../android-helpers';
-import log from '../logger';
-import { util } from 'appium-support';
 
-const swipeStepsPerSec = 28;
 
 let commands = {}, helpers = {}, extensions = {};
 
 commands.doSwipe = async function(swipeOpts) {
   return await this.uiautomator2.jwproxy.command(`/touch/perform`,'POST',{swipeOpts});
-}
+};
+
+commands.doDrag = async function (dragOpts) {
+  return await this.uiautomator2.jwproxy.command(`/touch/drag`,'POST', dragOpts);
```

Comment:
Are we passing in something different here than in `doSwipe`? It seems strange to have the `swipeOpts` put into an object, while `dragOpts` is not. Normalizing things would be good.